docs/modules/operation/pages/geographical-map/introduction.adoc

```diff
@@ -0,0 +1,36 @@
+# Geographical Maps
+
+OpenNMS includes a geographical map which can show nodes with address asset information on a world map, including dynamic alarm status.
+
+The geographical map is located in the `Maps` menu or by going to the page of a node that has an address and/or longitude/latitude defined in its asset fields, and following the `View in Geographical Map` link.
+
+## Markers and Clusters
+
+The geographical map displays each node that has an associated longitude and latitude in the database. 
+If a node does not have coordinates recorded, update the requisition to include address asset fields and verify the xref:admin/geocoder.adoc[geocoder service] is configured and enabled.
+
+The geographical map automatically clusters nodes into groups if they are close together. 
+A marker with a number in it indicates a cluster and the number of nodes in in it.
+Hover over a cluster to show the contents of the nodes it contains. 
+Click on a marker to show the summary of the nodes it contains.
+
+## Status
+
+A node's marker displays as the color associated with the most severe alarm for that node. 
+A cluster's marker displays as the color of the highest severity of the nodes the marker contains.
+
+
+*Normal*
+image::geographical-map/severity_normal.png[width=25,height=41]
```

Comment:
For some reason, these images are not showing up when I build the docs. I've tried troubleshooting, but nothing seems to work.

Reply:
figured it out - antora doesn't like how the name and image:: render.  review notes with a fix incoming.

Reply:
![image](https://user-images.githubusercontent.com/26419758/165155451-a4471e85-0ff5-46ab-b110-42553518ed61.png)